webrender/Cargo.toml

```diff
@@ -17,7 +17,7 @@ app_units = "0.4"
 bincode = "1.0.0-alpha6"
 bit-set = "0.4"
 byteorder = "1.0"
-euclid = "0.11.2"
+euclid = ">= 0.11, < 0.14"
```

Comment:
Well I didn't change any code so unless you're reexporting euclid somewhere I don't think that this is a breaking change.